scripts/automation/Radius/Start-RadiusDeployment.ps1

```diff
@@ -5,22 +5,25 @@ Connect-JCOnline $JCAPIKEY -force
 ################################################################################
 # Do not modify below
 ################################################################################
+# set script root
+$global:JCScriptRoot = $PSScriptRoot
+
 # Import the functions
-Import-Module "$psscriptroot/RadiusCertFunctions.ps1" -Force
+Import-Module "$JCScriptRoot/Functions/JCRadiusCertDeployment.psm1" -DisableNameChecking -Force
 
 # Show user selection
 do {
     Show-RadiusMainMenu
     $selection = Read-Host "Please make a selection"
     switch ($selection) {
         '1' {
-            . "$psscriptroot/Generate-RootCert.ps1"
+            . "$JCScriptRoot/Functions/Public/Generate-RootCert.ps1"
```

Comment:
Can't we just call `Generate-RootCert.ps1` here now that you've imported the module?

Reply:
The .psm1 file only imports the individual functions, whereas these .ps1 files act more like a main rather than individual functions